elections/README.md

```diff
@@ -0,0 +1,116 @@
+# Kubernetes Elections
+
+Welcome to the Kubernetes Elections area.  This directory defines the Elections
+Subproject, including its products of the annual [Steering Committee election]
+and elections-as-a-service for our community.  Elections is a subproject of
+[Contributor Experience] SIG.
+
+## Steering Elections
+
+The Elections subproject is responsible for the annual [Steerting Committee election],
+which is directly administered by the Election Officers.
+
+* If you are looking to vote in the steering election, go to the [elections app].
+* If you are looking for detailed information on the steering elections,
+  visit [Steering Committee election] page.
+
+## How to Request an Election for your SIG/WG
+
+All teams in Kubernetes may use our [elections app] to run preference elections
+for their team.  See the instructions on [requesting an election].
+
+## The Elections Subproject
+
+The Elections Subproject is a team within Kubernetes [SIG Contributor Experience]
+with the following responsibilities:
+
+* Maintain and update the elections documentation and messaging templates
+* Assist K8s-Infra team in maintaining the elections sofware and service
+* Assisting and approving SIGs/WGs in running minor elections
+* Recommending a slate of Election Officers for each Steering Election
+
+### Members
+
+Anyone can contribute to the elections subproject. In our [OWNERS file] you
+can find the list of our current approvers and reviewers.
+
+Approvers and reviewers are also responsible for the Elections roadmap, 
+maintenance, and security.  As such, new approvers must be approved by
+the [Contributor Experience] chairs or by the Steering Committee.
+
+### Communications
+
+The elections subproject can be reached by the following mechanisms:
+
+* tagging area/elections in issues or PRs in the Kubernetes/Community repo
+* #sig-contribex channel in Kubernetes slack
+* The weekly [Contributor Experience] meeting
```

Comment:
Technically the zoom one we have on the calendar is biweekly...thinking about it as a follow up item we should better document the slack standup one we host on alternate weeks.

Reply:
Heck, we need to make the slack standup a real thing.  That's on my TODO list as chair, but likely not until August.